### PR TITLE
Highlight selected credential

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the new profile
- toggle open a model
- click on the credential name
- it should take you to the credential list and the cred you selected should be highlighted

**Comment by huwshimi** [comment]
[highlight the selected cred](https://trello.com/c/LeMAtQx3/218-highlight-the-selected-cred)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8429/

File: jujugui/static/gui/src/app/components/profile/profile.js [hunk 1]
```diff
@@ -125,13 +126,19 @@ class Profile extends React.Component {
     return "credentials".
     @returns {String} The active base section of the URL.
   */
-  _getActiveBase() {
-    return (this.props.activeSection || '').split('/')[0];
+  _getProfileURL() {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/profile/profile.js]: Should add a test for this, yeah?

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8440/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8441/

**Pull request merged**